src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -200,7 +200,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     //   appended to this vector.
     // Return Value:
     // - a deserialized ActionAndArgs corresponding to the values in json, or
-    //   null if we failed to deserialize an action.
+    //   an "invalid" action if we failed to deserialize an action.
```

Comment:
note to self: this was used in two places:
```
src\cascadia\TerminalSettingsModel\Command.cpp:
  187:                 auto actionAndArgs = ActionAndArgs::FromJson(actionJson, warnings);

src\cascadia\TerminalSettingsModel\KeyMappingSerialization.cpp:
  134:             auto actionAndArgs = ActionAndArgs::FromJson(commandVal, warnings);

```
make sure those both do the right thing with Invalid instead of null now

Reply:
Now there's only 1 spot (`Command`)! It was fine before, but a little redundant. Pushing a minor change to make it less redundant and leverage that `FromJson` can return an invalid action instead of nullptr.